src/jit/graph.c

```diff
@@ -529,6 +529,8 @@ static MVMint32 consume_invoke(MVMThreadContext *tc, MVMJitGraph *jg,
 
             body = MVM_nativecall_get_nc_body(tc, object_facts->value.o);
             nc_jg = MVM_nativecall_jit_graph_for_caller_code(tc, iter->graph, body, restype, dst, arg_ins);
+            if (!body->entry_point) /* Check it only after we've used it to avoid a race condition */
```

Comment:
I'm a little confused by this comment. I think perhaps the 'only' should start the sentence, assuming you mean something like "The only reason we're checking this after already having used it is to avoid a race condition". Or do you mean "To avoid a race condition, this must only be checked after we've used it"?

Reply:
I updated the comment and the commit. Is it more clear now?